protected/humhub/modules/user/permissions/CanMention.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2016 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\user\permissions;
+
+use Yii;
+use humhub\modules\user\models\User;
+
+/**
+ * Can Mention Permission
+ */
+class CanMention extends \humhub\libs\BasePermission
+{
+
+    /**
+     * @inheritdoc
+     */
+    public $defaultAllowedGroups = [
+        User::USERGROUP_SELF,
+        User::USERGROUP_USER,
+        User::USERGROUP_FRIEND
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    protected $fixedGroups = [
+        User::USERGROUP_SELF,
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($config = [])
+    {
+        parent::__construct($config);
+        $this->title = \Yii::t('UserModule.base', 'Mentioning');
+        $this->description = \Yii::t('UserModule.base', 'Allows users to @mention you. Select "Deny" to allow only friends to mention you.');
```

Comment:
@Semir1212 Can you please check the text?

Reply:
To be honest, using `mention` is enough instead of using `@mention`, but I can see why it was used, so not only for the user to know how to use the function but also not to confuse community members who aren't used to using a mentioning function.

### Suggestion
`Allows other users to mention you, select "Deny" to allow only friends to mention you.`

Reply:
Looks good @ArchBlood @luke-